### PR TITLE
Audio gain variance

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

File: src/audio.rs [hunk 1]
```diff
@@ -80,7 +86,10 @@ impl Audio {
     ///
     /// The source can be created via [AudioSource::new] and modified by builders on [AudioSource]
     /// (like [AudioSource::looping]).
-    pub fn play(&mut self, source: AudioSource) {
+    pub fn play(&mut self, mut source: AudioSource) {
+        if source.gain_variance != 0.0 {
+            source.gain += random::Uniform.between(-source.gain_variance, source.gain_variance);
+        }
```
Comment (FredTheDino) [inline src/audio.rs:92]: Nice!

Maybe check if the gain is 0? Or can you change the gain as it's playing?

File: src/random.rs [hunk 2]
```diff
@@ -70,7 +75,7 @@ impl Distribute for ThreeDice {
     }
 }
 
-/// Biased towards 0. Looks like 1/x.
```
Comment (FredTheDino) [inline src/random.rs:73]: +1 for updating comments!

**Review by FredTheDino — APPROVED** [review]
Lgtm

File: src/audio.rs (hunk 1 above)
Comment (sornas) [reply src/audio.rs:92]: There is no way to change the gain currently. That should be part of a bigger "AudioID" PR.

**Pull request merged**